src/common/redis_module/ray_redis_module.cc

```diff
@@ -911,12 +911,24 @@ int TaskTableWrite(RedisModuleCtx *ctx,
                    RedisModuleString *state,
                    RedisModuleString *local_scheduler_id,
                    RedisModuleString *execution_dependencies,
-                   RedisModuleString *task_spec) {
+                   RedisModuleString *task_spec,
+                   RedisModuleString *spillback_count) {
   /* Extract the scheduling state. */
   long long state_value;
   if (RedisModule_StringToLongLong(state, &state_value) != REDISMODULE_OK) {
     return RedisModule_ReplyWithError(ctx, "scheduling state must be integer");
   }
+
+  long long spillback_count_value = 0;
+  if (spillback_count == NULL) {
+    /* TODO(atumanov): Note that this should not be possible, interim measure
+     * for task table update redis command. */
+    spillback_count = RedisModule_CreateStringFromLongLong(ctx,
```

Comment:
If we create a string, it probably needs to get freed with `RedisModule_FreeString` (but only if we go through this code path).

See the entry for RM_CreateStringFromLongLong at https://redis-module-redoc.readthedocs.io/en/latest/API/.

Reply:
Later we should look into using Redis's automatic memory management feature. https://redis-module-redoc.readthedocs.io/en/latest/

Reply:
actually, it was an interim plug, while the task_table_update wasn't ready. This code should never be called. I'll remove it.